### PR TITLE
Content update 3

### PR DESCRIPTION
**Pull request by kelvinkipruto**
This PR implements requested content update

### CODE REVIEW

File: src/app/crisis-helpline/crisis-helpline.component.html [hunk 1]
```diff
@@ -36,7 +36,7 @@ <h1 class="title">Crisis Helpline</h1>
     </div>
 
     <div class="item-1">
-      <h2>Lorem Ipsum Lorem Ipsum</h2>
+      <!-- <h2>Lorem Ipsum Lorem Ipsum</h2> -->
```
Comment (kilemensi) [inline src/app/crisis-helpline/crisis-helpline.component.html:39]: Should we remove this or what?

File: src/app/welcome-page/welcome-page.component.html [hunk 2]
```diff
@@ -53,9 +53,9 @@ <h1>
       <h2><span> Document the evidence,</span> unmask the trolls</h2>
 
       <p>
-        Use FeedShield to flag abusive content targeting you, so you can report the
-        abuse to the platforms or get our forensic investigators to track the
-        culprits.
+        Find out how to use FeedShield to flag abusive content targeting you, so you report the abuse to the platforms
+        or get
+        our forensic investigators to track the culprits.
```
Comment (kilemensi) [inline src/app/welcome-page/welcome-page.component.html:58]: This doesn't read right. Feels like a "can" is missing somewhere. Ping me on the source.

**Review by kilemensi — COMMENTED** [review]
👍🏽

File: src/app/crisis-helpline/crisis-helpline.component.html (hunk 1 above)
Comment (kelvinkipruto) [reply src/app/crisis-helpline/crisis-helpline.component.html:39]: I think we should remove it.

Comment (kilemensi) [reply src/app/crisis-helpline/crisis-helpline.component.html:39]:
```suggestion
      <h2>Virtual Emergency Support</h2>
```

Nah, I think the heading here is supposed to match the side-menu.

File: src/app/footer/footer.component.html [hunk 3]
```diff
@@ -22,10 +22,9 @@
       </a>
 
       <p class="summary">
-        FeedShield is a free tool specifically designed for Twitter users who face
-        online abuse on social media. The application syncs with a user’s
-        Twitter account and flags abusive content, thus limiting exposure to
-        targeted harassment.
+        Trolls use online violence to silence civic watchdogs. FeedShield is a free tool powered by Jigsaw and
+        implemented by
+        Code for Africa (CfA) to help Twitter users document and manage online harassment and abuse
```
Comment (kilemensi) [inline src/app/footer/footer.component.html:27]:
```suggestion
        Code for Africa (CfA) to help Twitter users document and manage online harassment and abuse.
```

**Review by kilemensi — APPROVED** [review]
🚀

**Pull request merged**